storage/SystemInfo/CmdDmraid.cc

```diff
@@ -80,22 +81,22 @@ namespace storage
 
 	for (const_iterator it = data.begin(); it != data.end(); ++it)
 	    y2mil("data[" << it->first << "] -> controller:" << it->second.controller <<
-		  " raidtype:" << it->second.raidtype << " devices:" << it->second.devices);
+		  " raid-type:" << it->second.raid_type << " devices:" << it->second.devices);
     }
 
 
-    list<string>
-    CmdDmraid::getEntries() const
+    vector<string>
+    CmdDmraid::get_entries() const
     {
-	list<string> ret;
+	vector<string> ret;
 	for (const_iterator it = data.begin(); it != data.end(); ++it)
 	    ret.push_back(it->first);
 	return ret;
     }
 
 
     bool
-    CmdDmraid::getEntry(const string& name, Entry& entry) const
+    CmdDmraid::get_entry(const string& name, Entry& entry) const
```

Comment:
I agree that the function interface is not nice. I will change it to return ```const Entry&``` for all currently used ```get_entry()``` functions in SystemInfo.